brk.c

```diff
@@ -86,7 +81,11 @@ UK_LLSYSCALL_R_DEFINE(void *, brk, void *, addr)
 	zeroed = addr;
 
```

Comment:
You can remove the blank lines between these assignments

Reply:
Done,